src/register/register.ts

```diff
@@ -70,7 +77,8 @@ export class Register {
     };
   }
 
-  public static putByKey(content: string | string[], register = '"', registerMode = RegisterMode.FigureItOutFromCurrentMode): void {
+  public static putByKey(content: string | string[] | RecordedState, register = '"',
```

Comment:
We should probably do `type RegisterContent = string | string[] | RecordedState` at this point.